bin/download_minister.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+import unicodecsv
+import requests
+import json
+import sys
+import os.path
+from lxml import etree
+from operator import itemgetter
+from datetime import date
+
+OUTPUT_FILE = os.path.join(os.path.split(__file__)[0],
+                           '../ckanext/canada/tables/choices/minister.json')
+lang_codes = ["en", "fr"]
+
+
+def get_filter_output():
+    """
+    Create a consolidated list of choices for ministers
+    """
+    existing_list = {}
+
+    # fetch existing list, if it exists
+    if os.path.isfile(OUTPUT_FILE):
+        with open(OUTPUT_FILE) as json_file:
+            try:
+                existing_list = json.load(json_file)
+            except ValueError as e:
+                return False
+        json_file.close()
+
+    # generate a list of choices from external source
+    choices = download_xml_from_source()
+
+    # if existing list does not exist
+    if not existing_list and choices:
+        return set_status(choices)
+    # if crawled choices list does not exist
+    elif existing_list and not choices:
+        return set_status(existing_list)
+    # if both exist, compare both lists for changes
+    elif existing_list and choices:
+        for row in existing_list:
+            if row in choices:
```

Comment:
instead of looping over both lists can you turn one into a dict or set and do a lookup?